commit ba8ae56d6836307cc7d36698173d3f2bcae12984 (refs/changes/99/899/2)
Author: David Pacheco <dap@joyent.com>
Date:   2016-11-14T15:31:43-08:00 (2 years, 11 months ago)
    
    TOOLS-1613 cr.joyent.us needs to work around gerrit issue 3885
    Reviewed by: Alex Wilson <alex.wilson@joyent.com>
    Approved by: Alex Wilson <alex.wilson@joyent.com>

diff --git a/images/appserver/etc/gerrit.config.base b/images/appserver/etc/gerrit.config.base
index ef22120..79b5d2f 100644
--- a/images/appserver/etc/gerrit.config.base
+++ b/images/appserver/etc/gerrit.config.base
@@ -16,6 +16,7 @@
 
 [index]
 	type = LUCENE
+	threads = 1
 
 [auth]
 	type = OAUTH
